client/app/bundles/comments/components/CommentScreen/CommentScreen.scss

```diff
@@ -0,0 +1,17 @@
+.notification {
+  padding: 1em 1.5em;
+}
+
+.logo {
+  background: url(./images/railsonmaui.png) no-repeat left bottom;
```

Comment:
I think we should point out that we can access relative paths in the CSS modules. **Very cool!**